controller/main.go

```diff
@@ -172,7 +171,7 @@ func main() {
 			PoolChanged:    c.SetPools,
 		},
 		ValidateConfig:      validation,
-		EnableWebhook:       *enableWebhook,
+		EnableWebhook:       true,
```

Comment:
I'd get rid of the field and all the consequent logic, since we can't set it to false

Reply:
Discussed offline, it's needed as the speaker must not enable the webhooks.

Reply:
if we don't set if both the controller and speaker enable webhooks.